### PR TITLE
Finished React week1 HW

### PR DESCRIPTION
**Pull request by Enqira**

### CODE REVIEW

File: react/week1/todo-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import TodoList from "./List"
+
+function App() {
+  const Todo = ({ title, deadLine }) => {
```
Comment (MaherHussain) [inline react/week1/todo-app/src/App.js:5]: hi, good job. but  this function should be out of App function

File: react/week1/todo-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    return (
+      <li>
+        <h2>{title}</h2>
+        <h3>{deadLine}</h3>
+      </li>
+    )
+  }
+
+  const TodoUl = () => {
```
Comment (MaherHussain) [inline react/week1/todo-app/src/App.js:14]: same here .

File: react/week1/todo-app/src/App.js (hunk 1 above)
Comment (Enqira) [reply react/week1/todo-app/src/App.js:5]: Ok thank you, I will fix that :)

**Review by MaherHussain — APPROVED** [review]

**Comment by MaherHussain** [comment]
feel free to  merge it

**Pull request merged**